src/pages/announcement/announcementPage.js

```diff
@@ -70,13 +73,15 @@ const AnnouncementContainer = styled.div`
     border-bottom: 3px #07645C solid;
     color: #07645C;
     font-size: 16px;
-    margin: 10px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-right: 40px;
     font-style: Open Sans;  
```

Comment:
Same issue here too.

```suggestion
    font-family: "Open Sans";
```